server/webapp/WEB-INF/rails.new/app/presenters/api_v4/shared/config_origin/config_repo_origin_representer.rb

```diff
@@ -28,7 +28,7 @@ def type
         end
 
         def repo
-          material = ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.new(config_repo.getMaterial).to_hash(url_builder: self)
+          material = ApiV5::Admin::Pipelines::Materials::MaterialRepresenter.new(config_repo.getMaterial).to_hash(url_builder: self)
```

Comment:
Isn't this representer used by the pipeline config api v4 alone? config repo representation is only present for environments and pipelines. If there is no other representer using api_v4 config repo origin representer, you can get rid of this as well?